app/controllers/orgs/rosters_controller.rb

```diff
@@ -228,6 +311,89 @@ def get_user_to_group_hash(grouping)
 
       mapping
     end
+
+    # Returns name of the linked google course to current organization (for syncing rosters)
+    def current_organization_google_course_name
+      return nil if current_organization.google_course_id.nil?
```

Comment:
If you find yourself writing `.nil?` in a conditional, there's usually a cleaner way to write it 😅 

```suggestion
     return unless current_organization.google_course_id
```